code/game/objects/items/devices/laserpointer.dm

```diff
@@ -120,8 +120,11 @@
 		log_combat(user, S, "shone in the sensors", src)
 		//chance to actually hit the eyes depends on internal component
 		if(prob(effectchance * diode.rating))
-			S.flash_act(affect_silicon = 1)
-			S.Paralyze(rand(100,200))
+			var/mob/living/silicon/robot/R = S
```

Comment:
Probably want to cut this line and change the var/mob/living/silicon/S = target on line 119 to var/mob/living/silicon/robot/S = target